meinberlin/assets/scss/components/_modal.scss

```diff
@@ -0,0 +1,60 @@
+// Note: Some styles for 'modals' are being set in variables.scss
+
+// overwriting bootstrap 'modal' classes
+.modal-content {
+    padding: 1.5 * $spacer;
+}
+
+.modal-body {
+    padding: 0;
+}
+
+.modal-header {
+    @extend .u-spacer-bottom;
+    padding: 0;
+
+    h1 {
+        margin: 0;
+        font-size: $font-size-xl;
+    }
+
+    button {
+        margin-left: auto;
+    }
+}
+
+.modal-footer {
+    @extend .u-spacer-top-half;
+    padding: 0;
+
+    button {
+        margin-top: 0;
+        margin-bottom: 0;
+
+        // case: two buttons
+        &:first-child {
+            margin-left: 0;
+        }
+
+        &:last-child {
+            margin-right: 0;
+        }
+    }
+}
+
+// stops modal moveing site content when opened
+.modal-open {
+    padding-right: 0 !important;
+}
+
+// used to preserve arbitrary* modal's appearances.
+// *arbitrary: e.g. modals that do not look and behave like our usual modals,
+// but also uses Bootstrap's modal classes.
+// example: blueprint tiles or mobile secondary filter navigation.
+.modal-content--rounded {
+    border-radius: $border-radius-lg;
+}
+
+.modal-content--padding {
+    padding: $spacer !important;
+}
```

Comment:
I think a lot of this can just be set in the variables https://getbootstrap.com/docs/5.0/components/modal/#variables

Reply:
yes true, I first had more of them in use. Unfortunately I could not make them working with our styles (especially the spacing/margin/padding).

I am even torn to move the one variable we use to css entirely, so we only have one source to change the modal styling, instead of two. What do you think?

Reply:
I think we should use the bootstrap variables as much as possible, did you find out what was overwritting them? just all the button spacing is already all there so we shouldn't need to re-add it if we can get it to work. Also `.modal-content--rounded` can be a utility as we can probably reuse it, also are we sure we want to have some modals with border radius and some without, doesn't that go against the consistency thing?